#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_mythweb_complaints.dpatch by Paul Andreassen
##
## All lines beginning with ## DP:' are a description of the patch.
## DP: Fix mythweb directory complaints.

@DPATCH@

Index: mythweb/includes/data_dir.php
===================================================================
--- a/mythweb/includes/data_dir.php	(revision 14883)
+++ b/mythweb/includes/data_dir.php	(working copy)
@@ -17,10 +17,13 @@
     if (!is_dir('data') && !mkdir('data', 0755)) {
         custom_error('Error creating the data directory. Please check permissions.');
     }
+
+    /*
     if (!is_writable('data')) {
         $process_user = posix_getpwuid(posix_geteuid());
         custom_error('data directory is not writable by '.$process_user['name'].'. Please check permissions.');
     }
+*/
 
 // New hard-coded cache directory
     define('cache_dir', 'data/cache');

diff -Nurp mythplugins/mythweb/modules/tv/handler.php mythplugins-t/mythweb/modules/tv/handler.php
--- mythplugins/mythweb/modules/tv/handler.php	2006-09-15 17:25:03.409605288 +1000
+++ mythplugins-t/mythweb/modules/tv/handler.php	2006-10-08 19:28:10.791587968 +1000
@@ -18,11 +18,13 @@
         custom_error('Error creating data/tv_icons: Please check permissions on the data directory.');
         exit;
     }
+/*
     if (!is_writable('data/tv_icons')) {
         $process_user = posix_getpwuid(posix_geteuid());
         custom_error('data/tv_icons directory is not writable by '.$process_user['name'].'. Please check permissions.');
         exit;
     }
+*/
 
 /**
  * @global  array   $GLOBALS['Categories']
